httpstan/services_stub.py

```diff
@@ -158,12 +158,14 @@ async def call(
 
     # if an exception has already occurred, grab relevant info messages, add as context
     exception = future.exception()
-    if exception:
-        num_context_messages = 4
-        info_messages_for_context = []
+    if exception and len(exception.args) == 1:
```

Comment:
Is there are any cases where the args could be larger than 1?

Reply:
It's certainly possible. Someone could throw `ValueError("foo", "bar")`.

What I really want to do is make sure it's the "Initialization failed" exception, but I don't want to use the string "Initialization failed" because that string is taken from Stan C++ output and could change in the future. For example, they could change it to "Initializing... failed" or something like that.

Reply:
Ok, makes sense